### PR TITLE
[vcpkg baseline] Fix baseline

### PR DESCRIPTION
**Pull request by JackBoosY**
Related: #7733 #11277 #11502 #11637 etc.

### CODE REVIEW

File: ports/msix/CONTROL [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Source: msix
-Version: 1.7
+Version: 1.7-1
 Build-Depends: xerces-c, zlib, openssl (!uwp&!windows), catch2
+Homepage: https://github.com/Microsoft/msix-packaging
```
Comment (c72578) [inline ports/msix/CONTROL:4]: Microsoft->microsoft

Comment (JackBoosY) [inline ports/msix/CONTROL:4]:
```suggestion
Homepage: https://github.com/microsoft/msix-packaging
```

**Review by BillyONeal — APPROVED** [review]

**Review by PhoebeHui — APPROVED** [review]

**Comment by dan-shaw** [comment]
Is this ready to merge?

**Comment by PhoebeHui** [comment]
@dan-shaw, yes, this PR is ready to merge.

**Pull request merged**